examples/ubuntu-vm-nfs-pv.yaml

```diff
@@ -0,0 +1,75 @@
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: nfs-dir-pvc
+spec:
+  accessModes:
+  - ReadWriteOnce
+  # Set Filesystem to Filesystem so the VM can consume this PVC
```

Comment:
```suggestion
  # Set volumeMode to Filesystem so the VM can consume this PVC
```

Reply:
sorry, it is my fault

Reply:
Don't be offended, it's not a problem :)